### ISSUE TITLE
"Your app may break" warning

### ISSUE THREAD

**Issue by taugustyn** [firebase/friendlyeats-web#37]
Seems like the code has to be updated to account for the breaking change:

```
firebase.js:1 [2018-09-17T13:07:12.836Z]  @firebase/firestore: Firestore (5.5.0): 
The behavior for Date objects stored in Firestore is going to change
AND YOUR APP MAY BREAK.
To hide this warning and ensure your app does not break, you need to add the
following code to your app before calling any other Cloud Firestore methods:

  const firestore = firebase.firestore();
  const settings = {/* your settings... */ timestampsInSnapshots: true};
  firestore.settings(settings);

With this change, timestamps stored in Cloud Firestore will be read back as
Firebase Timestamp objects instead of as system Date objects. So you will also
need to update code expecting a Date to instead expect a Timestamp. For example:

  // Old:
  const date = snapshot.get('created_at');
  // New:
  const timestamp = snapshot.get('created_at');
  const date = timestamp.toDate();

Please audit all existing usages of Date when you enable the new behavior. In a
future release, the behavior will change to the new behavior, so if you do not
follow these steps, YOUR APP MAY BREAK.
```

**Comment by samtstern** [comment]
This should have been fixed by #36, when did you clone the repo?

**Comment by taugustyn** [comment]
Few days ago, probably fixed by now.

**Comment by nicolasgarnier** [comment]
This was definitely filed before I merged #36
